js/fullWidthTreeView.js

```diff
@@ -218,12 +231,38 @@
     // a node is hovered to make it appear after node changes. It must
     // use the #fullwidth-treeview container to allow a higher
     // height than the node in multiple lines tooltips
+    var syncInitiated = {}; // Keep track of which syncs have been initiated
     var hoverNodeListener = function (e, data)
     {
+      // Configure tooltip
       $('a.jstree-anchor').tooltip({
         delay: 250,
         container: '#fullwidth-treeview'
       });
+
+      // Check sync of parent's children (if one hasn't yet been started)
+      var parent = data.node.parent;
+      var parentNode = $fwTreeView.jstree("get_json", parent);
+
+      if (parent != "#" && !(parent in syncInitiated) && ("href" in parentNode.a_attr)) {
```

Comment:
This if should also consider `!syncInitiated[parent]`, right?

Reply:
Good catch! I've changed it to delete the value rather than set it to false.